lib/rubocop/cop/rails/skips_model_validations.rb

```diff
@@ -100,7 +100,8 @@ def allowed_method?(node)
         end
 
         def forbidden_methods
-          obsolete_result = cop_config['Blacklist']
+          # TODO: Remove when RuboCop Rails 3 releases
```

Comment:
Can you add a period?

```suggestion
          # TODO: Remove when RuboCop Rails 3 releases.
```